README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion : recurrence or repetition,
```

Comment:
This is not specific enough.

Recursion is specifically when a method calls itself.